Swift/BRCrypto/BRCryptoAccount.swift

```diff
@@ -33,6 +33,15 @@ public final class Account {
         return Data (bytes: bytes!, count: bytesCount)
     }
 
+    internal init (core: BRCryptoAccount, uids: String, take: Bool) {
```

Comment:
Can we move the UIDS down to the C layer (tracked in a separate issue/PR, so as not to gate)? In keeping with our mantra of the C API being a full fledged implementation, I don't think we want anything tracked at the wrapper layer.

Reply:
Yes, and see CORE-421 (the similar, big trouble-maker).

Reply:
CORE-432 - UIDS into BRCryptoAccount